packages/unlock-js/src/__tests__/integration/single.js

```diff
@@ -18,11 +18,11 @@ import {
   versionEqualOrAbove,
 } from '../helpers/integration'
 
-const unlockVersion = process.env.UNLOCK_JS_TEST_RUN_UNLOCK_VERSION || 'v12'
+const unlockVersion = process.env.UNLOCK_JS_TEST_RUN_UNLOCK_VERSION || 'v11'
 const publicLockVersion =
-  process.env.UNLOCK_JS_TEST_RUN_PUBLIC_LOCK_VERSION || 'v13'
+  process.env.UNLOCK_JS_TEST_RUN_PUBLIC_LOCK_VERSION || 'v11'
 const testPath =
-  process.env.UNLOCK_JS_TEST_RUN_TEST_PATH || './lock/cancelAndRefund.js'
+  process.env.UNLOCK_JS_TEST_RUN_TEST_PATH || './lock/preparePurchaseKeys.js'
 
```

Comment:
??

Reply:
yes that is just a default value when running a single integration test locally... does not matter much really. It wont run on ci, just for local debug purpose